toddlamothe/website-analysis-web
# Added basic MVC structure/ moved report html to view file.

Test plan:
- run php build in server `php -S 127.0.0.1:8008 -t web/ web/front.php`
- visit page `http://localhost:8008/report` 
- make sure that you can see inputs for three competitors.
